pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -1166,4 +1222,36 @@ func validateFieldRef(fieldRef string, fldPath *field.Path) field.ErrorList {
 	return nil
 }
 
-var compiler = plugincel.NewCompiler(environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion()))
+var statelessCELCompiler = plugincel.NewCompiler(environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion()))
```

Comment:
Doc explaining that this validator is for use when variables are not accessible will help future us.

Reply:
Good point. Added.